templates/html/.partials/sidebar.html

```diff
@@ -44,7 +44,7 @@ <h2 class="text-xs uppercase text-grey mt-10 mb-4 font-thin">Operations</h2>
         </div>
         <div class="children">
           {% for channelName, channel in asyncapi.channels() %}
-             {% if channel.publish() and channel.publish().hasTags() and categorytags.name() === channel.publish().tags()[0].name() %}
+             {% if channel.publish() and channel.publish().hasTags() and channel.publish() | containTags(categorytags) %}
```

Comment:
I think `channel.hasPublish` would be more appropriate here, don't you think so?

```suggestion
             {% if channel.publish() and channel.publish().hasTags() and channel.hasPublish() | containTags(categorytags) %}
```

Reply:
I am using the publish operation to figure out if it contains the current tag in the loop, since the `hasPublish()` returns a boolean there would be tags for it to find. Unless this behaves differently then I think it does ^^

Reply:
Oh, you're right. Nevermind.